packages/api-aco/src/folder/onFolderBeforeDeleteHcms.hook.ts

```diff
@@ -0,0 +1,55 @@
+import WebinyError from "@webiny/error";
+import { CmsModel } from "@webiny/api-headless-cms/types";
+import { AcoContext } from "~/types";
+
+export const onFolderBeforeDeleteHcmsHook = (context: AcoContext) => {
+    context.aco.folder.onFolderBeforeDelete.subscribe(async ({ folder }) => {
+        try {
+            const { id, type } = folder;
+
+            const modelId = type.split(":")[1];
+
+            if (!modelId) {
+                return;
+            }
+
+            let model: CmsModel | null = null;
+            try {
+                model = await context.cms.getModel(modelId);
+
+                if (!model) {
+                    return;
+                }
+            } catch {
+                return;
+            }
+
+            const [entries] = await context.cms.listEntries(model, {
+                where: {
+                    latest: true,
+                    wbyAco_location: {
+                        folderId: id
+                    }
+                },
+                limit: 1
+            });
+
+            if (entries.length === 0) {
+                return;
+            }
+
+            throw new WebinyError(
+                "Error: delete all child folders and entries before proceeding.",
```

Comment:
Same here: remove the `Error:` prefix, and make it sentence case: `Delete all...`